### ISSUE TITLE
Diagrams: Load duration curve:

### ISSUE THREAD

**Issue by mstich0** [open-plan-tool/gui#209]
Add a load duration curve like:

![Image](https://user-images.githubusercontent.com/105202247/200308504-20d03435-4072-4786-8d18-a3067d14c0cf.png)

It should show the **output** flows (residual load) of **all** components (for all energy vectors)

**Comment by Bachibouzouk** [comment]
Without residual load and dual output component display both independently

**Comment by Bachibouzouk** [comment]
@mstich0, @c-moeller @FredericHirschmueller  - Shouldn't the load curve distinguish on the energy_vector? I.e. for CHP the electricity output should only be compared with other electricity productions in one graph and with other heat production in another graph. What about the output of the battery? Should it be added here?

**Comment by FredericHirschmueller** [comment]
Yes, the energy_vectors should be distinguished. I'm not even sure if a load duration curve for heat systems exist. I only know it for electrical systems, so it should only display the electricity vector. The battery should be added as well.

**Comment by mstich0** [comment]
Load duration for heating does exist and is used just the same way as for electricity. I´d agree that making 1 diagram per energy vector makes a lot of sense. I´d also agree to include the battery (output only)

**Comment by Bachibouzouk** [comment]
Load duration curves could also be displayed side by side (not stacked)

**Comment by Bachibouzouk** [comment]
This graphic still need some users inputs as to which assets of an energy system should be displayed

**Comment by Bachibouzouk** [comment]
It was decided to to first make the graphic not stacked
